challenge_16/views/app/show_data.html.erb

```diff
@@ -1,7 +1,10 @@
-<ul>
-<% @blogs.each do |blog| %>
-  <li>
-    <strong>Title: <%= blog.title %> </strong>, Content: <%= blog.content %>
-  </li>
+<%= tag.ul do %>
+  <% @blogs.each do |blog| %>
```

Comment:
Interesting, I never realized this was an "improper" way to write these ERB templates (ie. that the resulting raw HTML wouldn't be considered "correct")! 

Can you just clarify what the correct way to write this would be to have the correct amount of indentation? Is it only `<%` tags that should be indented?

Reply:
There’s no single correct way to do this, so it’s just a question of taste. If you wanted the HTML to have beautiful indentation for whatever reason, then you’d need to ignore the `<%` tags from the point of view of indentation (i.e. adding `<% … %>` … `<% end %>` tags around a section of HTML would not affect its indentation level), so each HTML tag ends up indented by the apparently-correct amount once all the ERB tags are removed during rendering. But to be completely clear, I don’t think anyone except me actually cares either way!